public/assets/img/provider_logos/bitss/style.css

```diff
@@ -0,0 +1,135 @@
+a,
+.preprint-form-section header {
+  color: #1C4063; }
+
+.preprint-header {
+  background-image: none;
+  background-color: #e4f3ff;
+  color: black;
+  padding-top: 100px; }
+  .preprint-header .provider-brand {
+    height: 140px; }
```

Comment:
Image sizes (and their related containers) should be defined outside of these CSS files as they are all standardized now. In a pinch we can do more simplification later; it doesn't have to be perfect to start, but the better it is, the easier it will be to maintain.